content/10.js/01.web3js/04.wallet.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Wallet
+description: Wallet capabilities from the Web3.js plugin for ZKsync
+tags: ["zksync", "sdks", "zksync era sdk", "ethereum", "era network", "web3.js", "web3.js plugin", "accounts", "account management", "wallet", "transaction signing"]
+---
+
+The Web3.js plugin for ZKsync exposes a [wallet](https://chainsafe.github.io/web3-plugin-zksync/classes/ZKSyncWallet.html)
+that allows developers to create, manage, and use ZKsync accounts.
+
+## Create a Wallet
+
+Once the Web3.js plugin for ZKsync has been [registered](./01.getting-started.md#register-the-plugin), it can be used to
+create a wallet.
+
+```ts
+import { Web3 } from "web3";
+import {
+  types,
+  Web3ZKsyncL2,
+  ZKsyncPlugin,
+  ZKsyncWallet,
+} from "web3-plugin-zksync";
+
+async function main() {
+  const web3: Web3 = new Web3(/* optional L1 provider */);
+  web3.registerPlugin(
+    new ZKsyncPlugin(
+      Web3ZKsyncL2.initWithDefaultProvider(types.Network.Sepolia),
+    ),
+  );
+  const zksync: ZKsyncPlugin = web3.ZKsync;
+
+  const PRIVATE_KEY: string = "<PRIVATE_KEY>";
+  const wallet: ZKsyncWallet = new zksync.Wallet(PRIVATE_KEY);
+
+  console.log("Wallet address:", await wallet.getAddress());
+}
+
+main()
+  .then(() => console.log("✅ Script executed successfully"))
+  .catch((error) => console.error(`❌ Error executing script: ${error}`));
+```
+
+## Deposit
+
+The [`ZKsyncWallet.deposit` method](https://chainsafe.github.io/web3-plugin-zksync/classes/ZKsyncWallet.html#deposit) is
+used to transfer ETH or any ERC-20 token from the wallet's L1 account to the target account on the ZKsync Era network.
+
+```ts
+import { TransactionReceipt, utils, Web3 } from "web3";
+import {
+  constants,
+  types,
+  Web3ZKsyncL2,
+  ZKsyncPlugin,
+  ZKsyncWallet,
+} from "web3-plugin-zksync";
+
+async function main() {
+  const web3: Web3 = new Web3(/* optional L1 provider */);
+  web3.registerPlugin(
+    new ZKsyncPlugin(
+      Web3ZKsyncL2.initWithDefaultProvider(types.Network.Sepolia),
+    ),
+  );
+  const zksync: ZKsyncPlugin = web3.ZKsync;
+
+  const PRIVATE_KEY: string = "<PRIVATE_KEY>";
+  const wallet: ZKsyncWallet = new zksync.Wallet(PRIVATE_KEY);
+  const senderL1BeginningBalance: bigint = await wallet.getBalanceL1();
+
+  const receiver: string = "<RECEIVER_ADDRESS>";
+  const receiverL2BeginningBalance: bigint =
+    await zksync.L2.getBalance(receiver);
+
+  const tx: types.PriorityOpResponse = await wallet.deposit({
+    token: constants.ETH_ADDRESS,
+    to: receiver,
+    amount: utils.toWei("0.00020", "ether"),
+    refundRecipient: wallet.getAddress(),
+  });
+
+  const receipt: TransactionReceipt = await tx.waitFinalize();
+  console.log("Transaction Hash:", receipt.transactionHash);
+
+  console.log(
+    "Sender Change In L1 Balance:",
+    (await wallet.getBalanceL1()) - senderL1BeginningBalance,
```

Comment:
Just curious, will this still work if there is no L1 provider defined in the `new Web3()`?

Reply:
Thank you for catching that 🙏🏻 I have updated the necessary examples to include an L1 provider.